index.bs

```diff
@@ -1553,13 +1553,19 @@ partial interface MLGraphBuilder {
 ### pooling operations ### {#api-mlgraphbuilder-pool2d}
 Compute a *mean*, *L2 norm*, or *max* reduction operation across all the elements within the moving window over the input tensor. See the description of each type of reduction in [[#api-mlgraphbuilder-reduce]].
 <script type=idl>
+enum MLRoundingType {
+  "floor",
+  "ceil"
+};
```

Comment:
See my comment on #198. Rounding mode for tensor size calculation should be done above WebNN, similar to how it is done in ONNX Runtime above DirectML.

Reply:
@wchao1115 , thanks for your comments on #198. 

However a caller may not calculate the tensor shape and rely on WebNN to do that, e.g. the OpenCV WebNN backend mentioned in #198. The [MLOperand](https://webmachinelearning.github.io/webnn/#api-mloperand) interface of WebNN doesn't allow query the shape. So if the caller doesn't infer the shape by itself, it would be hard to calculate the output shape with a desired rounding type from the input shape, especially when the input is an intermediate operand. That's why I propose to let the callers to just configure the rounding type instead of calculating the output shape at all by themselves. 

I suppose this would map ONNX `ceil_mode` well. What do you think?

Reply:
That's a fair point. The current WebNN design does allow dynamic shape inference, which would make it somewhat easier to call the API directly (since the caller needs not worry about correctly implementing shape inference for all cases themselves), but it does come at a cost of additional work on the implementer's side, and in theory some long-term maintenance cost of the API due to the additional policy that must be implemented for such a caller.

If we want to continue to allow dynamic shape inference in the WebNN API, then adding an optional output size's rounding mode would not be out of line. In that case, I would ask that we also add `outputSizes` so if the framework callers already calculate the output size themselves (most frameworks do), then they can just ignore the rounding mode.

Reply:
> In that case, I would suggest that we also add `outputSizes` so if the framework callers already calculate the output size themselves (most frameworks do), then they can just ignore the rounding mode.

It sounds good to me. 

@wchao1115 , according to DirectML backend implementation, how does pooling ops of DirectMLX, such as [`AveragePooling`](https://github.com/microsoft/DirectML/blob/master/Libraries/DirectMLX.h#L2061), support rounding mode? It calculates the output sizes inside and doesn't allow to configure neither rounding type nor output sizes.

Reply:
[DirectMLX](https://github.com/microsoft/DirectML/blob/master/Libraries/DirectMLX.h) is just a helper library to DirectML. It doesn't add any meaningful feature to it, just literally reduces typing and makes DirectML easier to access. The library also doesn't do shape inference at runtime, only at construction time, so if the framework that uses it support runtime shape inference, it needs to handle that before passing it down to DirectML or to its helper DirectMLX.

Reply:
Thanks @wchao1115 .

`outputSizes` is added into the latest commit. Please take another look.